packages/orbit-components/src/Wizard/Wizard.stories.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { number, array } from "@storybook/addon-knobs";
+import { action } from "@storybook/addon-actions";
+
+import Wizard, { WizardStep } from ".";
+
+storiesOf("Wizard", module).add(
```

Comment:
@silvenon could you also add a story with RTL version? 🙏

Reply:
Added a RTL story and fixed RTL issues, _however_ the text in the toggle button in mobile doesn't look right to me:

<img width="495" alt="Screen Shot 2020-10-15 at 12 45 48" src="https://user-images.githubusercontent.com/471278/96113357-608b3400-0ee4-11eb-9c71-87bcd4f4cc4f.png">

How come it says "of 5 4"…?

Reply:
This is okay, because `direction: rtl` can't work with normal latin text 🙂 
The browser is solving this automatically, we just need to worry about CSS, nothing more 😉